packages/core/src/Page/index.ts

```diff
@@ -549,6 +576,7 @@ export class Page {
       await this.outputPageHtml(content);
     } else {
       const vueSsrHtml = await pageVueServerRenderer.renderVuePage(compiledVuePage);
+      this.filterIconAssets(content, vueSsrHtml);
```

Comment:
For the issue of additional imports of other icons in your other test pages, I guess it's either because there are other components such as pageNav/siteNav/layout that contains those icons, or because ` this.filterIconAssets(content, vueSsrHtml);` is not ran for the line 575 ` if (process.env.TEST_MODE)` branch. If it's the latter, we need to adjust where we place this line of coded, and also the functionality of filterIconAssets

What do you think?

Reply:
Right, it solves the issue. Thanks for the catch!

Reply:
Actually, the stylesheets might not be filtered correctly for TEST_MODE. For example, in testGlyphiconInPage, the actual HTML of <panel> which contains glyphicon icons is not reflected in `content`. As a result, the glyphicon stylesheet is not loaded.

Reply:
hmm, this is because of vuessr as well? Can we address this by scanning through the compiledVuePage (or something else)?

Reply:
> Actually, the stylesheets might not be filtered correctly for TEST_MODE

You mean this is a bug?

Reply:
it's because TEST_MODE only outputs the pre vue-compiled HTML which consists of "unprocessed" vue components (e.g. `<panel>`,  `<modal>`). The actual HTML of these components might consist of icons. We can simply address this by removing the 'if' condition, i.e. regardless of TEST_MODE or not,  we will output the post vue-compiled HTML.

Reply:
I see. It's ok I'm fine with the stylesheets not entirely filtered for TEST_MODE.

Reply:
Hi @jmestxr, sorry I think we should handle the case of filtering for TEST_MODE. We don't have to output the post vue-compiled HTML for TEST_MODE, but we can always render it and give it for filtering. Something like:
```
    const vueSsrHtml = await pageVueServerRenderer.renderVuePage(compiledVuePage);
    this.filterIconAssets(content, vueSsrHtml);
    if (process.env.TEST_MODE) {
      await this.outputPageHtml(content);
    } else {
      await this.outputPageHtml(vueSsrHtml);
    }
```

What do you think?

With this I should expect to see glyphicon CSS import in packages/cli/test/functional/test_site/expected/testGlyphiconInPage.html

Reply:
@tlylt This makes sense and it works. Thanks!